rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -1086,7 +1086,7 @@ SecRule REQUEST_PROTOCOL "!@within %{tx.allowed_http_versions}" \
 #
 # Restrict file extension
 #
-SecRule REQUEST_BASENAME "@rx \.(.*)$" \
+SecRule REQUEST_BASENAME "@rx \.([^\.]+)$" \
```

Comment:
You don't need to escape the period inside a character class. Otherwise LGTM.

Reply:
thanks @fgsch I try to commit it without escape

Reply:
You probably want to rebase it to exclude all the other changes/commits too.

Reply:
@fgsch I think I've done with rebase. Could you check if it's ok?

Reply:
Perfect!